addon/styles/_variables.scss

```diff
@@ -6,6 +6,7 @@
 // Notification base
 // -------------------------
 
+$notification-top: 50px !default;
```

Comment:
https://robots.thoughtbot.com/sass-default !default basically allows safe overrides that can be specified before (or after) our theme defaults